### PR TITLE
Changes related to site aesthetics

### PR DESCRIPTION
**Pull request by Shoval-Ovadia**
- Changed the order of the links in the navigation bar - pickATrip_django_apps/templates/pickATrip_django_apps/base.html modified.
- Changed the URLs to valid ones in comments test-data - commenting_system/migrations/0002_test_data.py modified
- The user will be able to click on an image in the list of posts and reach the relevant post details - Post/templates/Post/postList.html modified.
- Changed the view of a single post to contain an image - Post/templates/Post/post_detail.html modified.

Because I changed the 'test data' files, before you check the changes - do the following steps:
- run vagrant halt
- delete 'db.sqlite3'
- vagrant up
- vagrant provision

### CODE REVIEW

File: pickATrip_django_apps/templates/pickATrip_django_apps/base.html [hunk 1]
```diff
@@ -41,14 +41,15 @@
         </li>
 		{% if user.is_authenticated %}
 		<li class="nav-item">
-          <a class="nav-link " aria-current="page" href={% url 'profile' %}>profile</a>
+          <a class="nav-link " aria-current="page" href={% url 'createPost' %}>Create new post</a>
         </li>
 		<li class="nav-item">
-          <a class="nav-link " aria-current="page" href={% url 'logout' %}>Logout</a>
+          <a class="nav-link " aria-current="page" href={% url 'profile' %}>profile</a>
```
Comment (Leeadar) [inline pickATrip_django_apps/templates/pickATrip_django_apps/base.html:47]: you can capitalize the 'profile' to 'Profile':
` <a class="nav-link " aria-current="page" href={% url 'profile' %}>Profile</a>`

File: Post/templates/Post/postList.html [hunk 2]
```diff
@@ -3,7 +3,8 @@
 
 	{% for post in posts %}
 		<h1>{{post.nameOfLocation}}</h1>
-		<img src= {{post.photoURL}} alt="photo of {{post.nameOfLocation}}" style="width:300px;height:300px;">
+		<a href="{% url 'post_detail' post.id %}">
```
Comment (maayanhd) [inline Post/templates/Post/postList.html:6]: very nice!

File: pickATrip_django_apps/templates/pickATrip_django_apps/base.html (hunk 1 above)
Comment (maayanhd) [inline pickATrip_django_apps/templates/pickATrip_django_apps/base.html:47]: maybe just 'Profile' using capital letter other than 'profile'

**Review by Leeadar — COMMENTED** [review]
In the base.html file, line 24:
 <a class="navbar-brand" href="#">
Please change to:
href={% url 'homepage' %}

File: pickATrip_django_apps/templates/pickATrip_django_apps/base.html [hunk 3]
```diff
@@ -41,14 +41,15 @@
...
         </li>
 		<li class="nav-item">
-          <a class="nav-link " aria-current="page" href={% url 'createPost' %}>Create new post</a>
+          <a class="nav-link " aria-current="page" href={% url 'logout' %}>Logout</a>
         </li>
+
```
Comment (maayanhd) [inline pickATrip_django_apps/templates/pickATrip_django_apps/base.html:52]: did you do this space or is it something to do with black formatting?

File: commenting_system/migrations/0002_test_data.py [hunk 4]
```diff
@@ -26,15 +26,15 @@ def generate_data(apps, schema_editor):
         Post_list = [
             Post(
                 nameOfPoster='Leead',
-                nameOfLocation='Eilat',
-                photoURL='test_1.com',
+                nameOfLocation='Sea of Galilee',
+                photoURL="https://www.shappo.co.il/Resize_Image.aspx?maxsize=400&img=/pictures/cards/big/36630.jpg",
```
Comment (maayanhd) [inline commenting_system/migrations/0002_test_data.py:30]: Are these photos working for you?

File: commenting_system/migrations/0002_test_data.py [hunk 5]
```diff
@@ -26,15 +26,15 @@ def generate_data(apps, schema_editor):
...
                 Description='Perfect!',
             ),
             Post(
                 nameOfPoster='Shoval',
-                nameOfLocation='Tel-Aviv',
-                photoURL='test_2.com',
-                Description='This is nice',
+                nameOfLocation="`En Yorqe`am",
+                photoURL="https://cdn1.sipurderech.co.il/1200x800_fit_90/1403218722_121.jpeg",
```
Comment (maayanhd) [inline commenting_system/migrations/0002_test_data.py:36]: this either is not working for me 
image

Comment (Shoval-Ovadia) [reply commenting_system/migrations/0002_test_data.py:36]: Did you follow the steps?
You need to delete the DB file.

**Review by maayanhd — APPROVED** [review]
Working like magic, great final touch soon to be merged hopfully

**Review by Leeadar — APPROVED** [review]
Looks fabulous!

**Review by nadavspinco — APPROVED** [review]

**Review by Yarboa — APPROVED** [review]

**Pull request merged**